components/compositing/compositor.rs

```diff
@@ -385,10 +385,6 @@ impl<Window: WindowMethods> IOCompositor<Window> {
                 return false
             }
 
-            (Msg::Exit, _) => {
-                self.start_shutting_down();
```

Comment:
I don't think this gets called anymore after these changes.

Reply:
[By appending `WindowEvent::Quit` here](https://github.com/servo/servo/pull/20865/files#diff-820b6f9bc7650b5463893408956097ffR309), `ompositor.maybe_start_shutting_down` ends up being called via [Servo's handling of `WindowEvent::Quit` here](https://github.com/gterzian/servo/blob/e5df6eb35b64b10c1cb020cde81f77e72961e77e/components/servo/lib.rs#L311), and `maybe_start_shutting_down` [ends up calling `start_shutting_down`](https://github.com/gterzian/servo/blob/e5df6eb35b64b10c1cb020cde81f77e72961e77e/components/compositing/compositor.rs#L352).
So the difference is that the compositor is shut down only after the unload steps have been run. If the embedder were to handle tabs or something similar, it wouldn't have to send a `WindowEvent::Quit` in case other tabs were still running...

Reply:
Ah missed that! This sounds reasonable.